Running test on mutant:
============================= test session starts ==============================
platform linux -- Python 3.8.19, pytest-8.3.3, pluggy-1.5.0
rootdir: /mnt/temp/inspect_mutant
collected 1 item

../test.py F

=================================== FAILURES ===================================
_____________________________________ test _____________________________________

    def test():
        formatter = __StringFormatter("")
        match = re.match(r"(a)(b)", "ab")
    
>       assert formatter._StringFormatter__ensure_right_space_only(match) == "a "
E       AssertionError: assert 'ab ' == 'a '
E         
E         - a 
E         + ab

../test.py:9: AssertionError
=========================== short test summary info ============================
FAILED ../test.py::test - AssertionError: assert 'ab ' == 'a '
============================== 1 failed in 0.04s ===============================
